jraft-core/src/main/java/com/alipay/sofa/jraft/rpc/impl/BoltRpcClient.java

```diff
@@ -57,7 +57,11 @@ public boolean init(final RpcOptions opts) {
         this.rpcClient.option(BoltClientOption.NETTY_FLUSH_CONSOLIDATION, true);
         this.rpcClient.initWriteBufferWaterMark(BoltRaftRpcFactory.CHANNEL_WRITE_BUF_LOW_WATER_MARK,
             BoltRaftRpcFactory.CHANNEL_WRITE_BUF_HIGH_WATER_MARK);
-        this.rpcClient.enableReconnectSwitch();
+        if (opts.isEnableBoltReconnect()) {
+            this.rpcClient.enableReconnectSwitch();
+        } else {
+            this.rpcClient.disableReconnectSwith();
+        }
```

Comment:
`enableReconnectSwitch` and `disableReconnectSwitch` are deprecated. It appears we can use `option(BoltClientOption.CONN_RECONNECT_SWITCH, opts.isEnableBoltReconnect())` instead.